docker/compose/docker-compose.yml

```diff
@@ -87,6 +87,7 @@ services:
     command: segmentstore
     environment:
       WAIT_FOR: bookie1:3181,bookie2:3182,bookie3:3183,hdfs:8020
+      TIER2_STORAGE: "HDFS"
```

Comment:
Out of curiosity, why didn't we need to specify tier 2 storage before, and now we do?

Reply:
My assumption is that this particular Docker compose file has not been used since the `TIER2_STORAGE` was introduced. @arvindkandhare can you confirm that?

Reply:
This is right. The assumption is that the default is `HDFS` but it was changed to `INMEMORY`. It is safer to explicitly mention it here.